go/vt/automation/testing_emit_echo_task.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, Google Inc. All rights reserved.
```

Comment:
The problem with these Testing* tasks is that they are referenced in the factory function which instantiates tasks from the protobuf-description:

https://github.com/youtube/vitess/commit/20c10dbe7d19c462935a16002da33550668e2e1d#diff-3a3658336f82b97340ba072879b56fd9R166 

Therefore, we cannot compile them as test code only.

If so: When running tests, we would have to extend the factory method to be aware of the Testing* tasks as well. Can you think of an easy way to do this?